src/app/app.module.ts

```diff
@@ -52,7 +54,9 @@ import { CategoriesModule } from './categories/categories.module';
     ExpensesListModule,
     Ng4LoadingSpinnerModule.forRoot(),
     ProductsModule,
+    CategoryFormModule,
     ProvidersModule,
+    AppRoutingModule,
```

Comment:
@BaidaaAlbzoor  AppRoutingModule, was already imported , and it must kept last one as we learned from routing issue